Projet-Web-Avance-A4/Developpeur
# add create and connexion + style

**How to test ?**

1) Go to this [page](http://localhost:3000) after launching this repository in Port 3000 and Microservices in Port 3001.
2) Create your account.
3) Try to connect, if it's right, you should be redirected to the download page. If not, an error should appear.

PS : You should not be able to create an account or connect if one of the required input is not filled. The style for it will be done later.
